frontend/src/app/main/ui/ds/foundations/heading.mdx

```diff
@@ -0,0 +1,86 @@
+import { Canvas, Meta } from "@storybook/blocks";
+import * as HeadingStories from "./heading.stories";
+
+<Meta of={HeadingStories} />
+
+# Headings
+
+We currently work with 3 scales depending on whether the typography
+is applied in the workspace, in the dashboard and access areas
+and in the communication modalities.
+
+This situation is something to be corrected in future improvements.
+
+## Usage
+
+### Standard text colours
+
+Typographic colours are used in text elements such as headers
+and body and in the various components that make up the tool...
+
+The colours used in typography are the Foreground colours but
+different colours can be applied in specific component applications
+with their own styles, such as buttons.
+
+As far as possible and as long as accessibility is not affected,
+we will use colour in typography as a way of hierarchising
+the content displayed using tone, built in luminance changes and
+size as differentiating elements between main and secondary text.
+
+### Status text colous
+
+Another possible application of colour is semantic, when we use text to convey some system state.
+
+Use the tokens:
+$color-text-success - Use as text colour to indicate success.
+$color-text-warning - Use as text colour to indicate a warning or caution.
+$color-text-error - Use as text colour to indicate an error.
+
+## Accesibility
+
+Typefaces should be sized for legibility and accessibility. 
+The minimum size for texts in Penpot will be 14px except 
+for exceptions based on the size of the components.
+
+## Technical notes
+
+This component accepts three props. Children, which refers to its content. 
```

Comment:
`children` is implied, no need to mention it (as well as `className` and all the delegated props we could pass).